broker/endpoints/Patient.js

```diff
@@ -135,8 +135,12 @@ exports.create = [
     return Patient.genderEnums[genderVal];
   }),
   body('Patient.*.birthDate', 'Please enter patient birth date').trim().isDate({format: 'MM/DD/YYYY'}),
-  body('Patient.*.race', 'Please enter patient race').trim().escape().isLength({min: 1}),
-  body('Patient.*.ethnicity', 'Please enter patient ethnicity').trim().escape().isLength({min: 1}),
+  body('Patient.*.race', 'Please enter patient race').trim().escape().custom((raceVal) => {
```

Comment:
Do we have to require race and ethnicity?

Reply:
Race and ethnicity are required fields by the registration app.